src/App.vue

```diff
@@ -384,6 +386,10 @@ export default class App extends Mixins(
       this.fileAndPayInvalidNameRequestDialog
     )
   }
+  /** The Fee Summary filing text for firm businesses. */
+  get filingLabelText (): string {
+    return this.isTypeFirm && 'Dissolution'
```

Comment:
It would be better to use a ternary expression or if-then-else because this may return `false`.

Since this getter is supposed to return a string, acceptable "empty" return values are `''` or `null`.

Reply:
fixed